docs/docs/noir/standard_library/meta/typed_expr.md

```diff
@@ -6,8 +6,14 @@ title: TypedExpr
 
 ## Methods
 
-### as_function_definition
+### get_type
 
 #include_code as_function_definition noir_stdlib/src/meta/typed_expr.nr rust
 
-If this expression refers to a function definitions, returns it. Otherwise returns `Option::none()`.
\ No newline at end of file
+If this expression refers to a function definitions, returns it. Otherwise returns `Option::none()`.
+
+### get_type
+
+#include_code get_type noir_stdlib/src/meta/typed_expr.nr rust
+
+Returns the type of the expression, if the expression could be resolved without errors.
```

Comment:
```suggestion
Returns the type of the expression, or `Option::none()` if there were errors when the expression was previously resolved.
```
Trying to be more explicit that the expression is already resolved. If not, users would have to worry about the scope `get_type` is called in.